client/src/interface/User.ts

```diff
@@ -1,6 +1,6 @@
 export interface IUser {
   email: string;
-  picture?: string;
+  picture: { url: string; etag: string };
```

Comment:
I did talk to you about `etag`. My only concern is you aren't using this property at all. I was telling you the `etag` was a potential solution to solving this problem of duplicate uploads. We don't need `etag` in this solution you provided. Perhaps some confusion

Reply:
It was already removed. I thought I had pushed the commit where I removed it.